orders/urls.py

```diff
@@ -0,0 +1,7 @@
+from django.urls import path
+
+from orders.views import OrderNowView
+
+urlpatterns = [
+    path('/ordernow', OrderNowView.as_view())
```

Comment:
```suggestion
    path('/now', OrderNowView.as_view())
```
그냥 now라고 해도 될 것 같아요.

Reply:
넵 알겠습니당~